src/macaron/slsa_analyzer/provenance/expectations/expectation_registry.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Was this `policy_registry.py` originally? Do we remove any part in this module except name changing?

Reply:
I removed the YAML and Datalog policy registrations, because they are not used as expectations anymore.